src/JsonApi/EventListener/TransformFieldsetsParametersListener.php

```diff
@@ -49,7 +49,7 @@ public function onKernelRequest(RequestEvent $event): void
             return;
         }
 
-        $includeParameter = explode(',', $includeParameter ?? '');
```

Comment:
I'm a bit afraid of this kind of change. Most of them are here for backward compatibility. Please double check that it's really not needed before changing the code in `src/` (it's ok to modify`tests/`)

Reply:
From phpstan and my reading of the previous condition, we are already checking: 
`!$includeParameter`
and
`(!\is_string($includeParameter))`
So I was thinking that $includeParameter can not be null in explode